### DISCUSSION TITLE
Doubts in PNG Image Resize

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#5448] [Help]
I need to resize the png image to different dimension 
Since png is  lossless compression format , is it okay to resize from already resized image of that original image instead of original image ? 

Ex: If I need to resize the image greater than 2000X2000 to three different Dimensions(1500X1500, 1000X1000, 500X500)
For resizing the original image to 500X500 can use 1000X1000 image as source image instead of original image as source image?

Kindly revert me if the above assumption is wrong

**Comment by fmw42** [comment]
Resize from the original as each resize may degraded the image slightly due to interpolation. You can resize to 3 sizes all in one command using parenthesis processing. See for example https://imagemagick.org/Usage/files/#write
